docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md

```diff
@@ -351,3 +352,28 @@ Deploy clusters using [quicklabs](https://quicklab.upshift.redhat.com/). Same us
 #### OpenShift CI Native Debugging
 
 Testing prow job specifics secrets, vars, and things like that can be done through a PR to the release repo, but a cluster deployment is not needed. So if you need to test that a secret that you are using is going to show up properly you can submit a PR strictly for debugging purposes to run the prow job and discover the necessary things that you need to do. Once you've solved for that small problem you can then integrate the solution back into the larger PR for your scenario.
+
+### Accessing the Test Cluster
+
+If your scenario provisions a test cluster using a workflow like `ipi-aws`, follow these steps to access the test cluster's console:
+
+1. Start a rehearsal job using `/pj-rehearse`
+2. Open the rehearsal job's logs and find the build cluster URL under the "Build Logs" header and click on it to access the build cluster
```

Comment:
Steps 2-4 is one way to find the console but I think its probably the hardest.

We can get the console url two different ways that we'll probably want to mention here.
1. This only works if you have access to oc client `oc whoami --show-console`
2. `cat $SHARED_DIR/console.url`

Reply:
@ascerra That is to get to the Prow/Build cluster console from the rehearsal logs (right after you click "details" in GitHub next to your rehearsal job).

But, step 5 is harder and I will share those.

Reply:
I have updated the docs to use those methods instead, thanks!